recipes/approvaltests.cpp/all/conandata.yml

```diff
@@ -154,3 +154,13 @@ sources:
       sha256: adcc8ac09777f62eda5dcb721d6c1ebc9f22a621997c761920c9d3858c4d3eff
     - url: "https://raw.githubusercontent.com/approvals/ApprovalTests.cpp/v.10.8.0/LICENSE"
       sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
+  10.9.0:
```

Comment:
Can this version be removed?

Reply:
OK - at first I thought it would be confusing that there was a break in the sequence - and the issue with 10.9.0 will affect hardly any users... but on reflection, I do think it's a good idea.

Reply:
Now done - thanks for the suggestion